src/main/resources/org/jfrog/hudson/ArtifactoryServer/config.jelly

```diff
@@ -0,0 +1,30 @@
+<?jelly escape-by-default='true'?>
+
+<jelly:jelly xmlns:jelly="jelly:core" xmlns:f="/lib/form">
+  <f:entry field="id" title="Server ID" help="/plugin/artifactory/help/ArtifactoryBuilder/help-serverId.html">
```

Comment:
I suggest to stick to the default naming convention for help files. Then the `help` attribute is not required anymore.

Reply:
Copy paste for now 😅

Reply:
Fixed, though could add some more help files